src/pages/_id/index.vue

```diff
@@ -35,8 +35,8 @@
           </div>
           <UserStatsPortfolio
             class="grid grid-cols-2 cursor-default gap-x-8 gap-y-4 text-medium-gray"
-            :collected-items="collectedNFTs"
-            :created-class-ids="sellingNFTClassIds"
+            :collected-items="collectedClassIds"
```

Comment:
Suggest to rename the prop name too
```suggestion
            :collected-class-ids="collectedClassIds"
```